process faults {3: ({1}, 3460319281)}
network faults {4: [{0, 1, 4}, {2}, {3}, {5}, {6}]}
{6: ("D7559DA81E213FB0353FA230173782A92592BAC1F37A44BBECD03B1852872241", {2, 4, 1, 3, 5, 0, 6}), 9: ("7DD0A74ED19C3D7C2BFF1A2E1CD0DF1E52F1F1F980892E66C2DD6D0764446059", {0, 6, 5, 1, 2, 3, 4}), 5: ("49FB6675E12D7B3C658427232B32C6BDAD1278CB34D6A716C8ADE82EB312A477", {4, 1, 2, 5, 6, 0, 3}), 8: ("F12617B4D3721EFCB6316D06A624AACCEB726CD6E6B41096904F7BE45C89C728", {6, 5, 0, 2, 1, 3, 4}), 10: ("D57B400F4BA5A3CD7CE37AB04D57E929B65D429FBCD3A9A394A9ED7ACBF4F912", {2, 0, 5, 3, 4, 6, 1}), 7: ("CB836AE0087FF53CFA018BB020B9FF7BB713C0BE9348058BB440F743E390F23B", {3, 1, 2, 5, 6, 4, 0})}
7
reason: all committed
done!
